tests/integration/test_storage.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for `ebr_trackerbot` package."""
+
+import sys
+import os
+
+sys.path.append("ebr_trackerbot")
+sys.path.append("ebr_trackerbot/storage")
+
+from bot import config
+from state import STATE
+import pytest
+import sqlite
+import odbc
+import tempfile
+
+
+def delete_fixtures():
+    if os.path.exists(tempfile.gettempdir() + "/data.db"):
+        os.unlink(tempfile.gettempdir() + "/data.db")
+    if os.path.exists(tempfile.gettempdir() + "/data-odbc.db"):
+        os.unlink(tempfile.gettempdir() + "/data-odbc.db")
+
+
+def reset_fixtures():
+    delete_fixtures()
+    open(tempfile.gettempdir() + "/data-odbc.db", "a").close()
+
+
+def test_storage():
+    reset_fixtures()
+    config["odbc_connection_string"] = "DRIVER={SQLITE3};DATABASE=" + tempfile.gettempdir() + "/data-odbc.db"
+    modules = ["odbc", "sqlite"]
```

Comment:
Maybe instead it should be a parameterized test?

Reply:
Ok